osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/model/DlmsAutoAnswer.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.dlmsobjectconfig.model;
+
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.dlmsobjectconfig.DlmsObjectType;
+
+public class DlmsAutoAnswer extends DlmsObject {
```

Comment:
define what this class does/means

Reply:
These classes (all derived from DlmsObject) are for representing all DLMS objects in the DlmsObjectConfigs. Each object has a DLMS class Id. If necessary, a class could have additional attributes or methods for that specific DLMS class (e.g. see DlmsRegister class).

Defining in the code what an AutoAnswer class actually is for, is not necessary (this is defined in the DLMS standard).

Reply:
ok clear